tests/test_gear_advisor.py

```
@@ -9,13 +9,148 @@
+def _make_chroma_result(docs, metas, indices=None):
+    """Build a ChromaDB-style query result dict."""
+    if indices is not None:
+        docs = [docs[i] for i in indices]
+        metas = [metas[i] for i in indices]
+    return {"documents": [docs], "metadatas": [metas]}
+
+
+def _fake_query_shoes(query_texts, n_results=10, where=None):
+    """Return filtered shoe results based on where clause."""
+    docs, metas = [], []
+    for d, m in zip(_FAKE_SHOE_DOCS, _FAKE_SHOE_METAS):
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)